app/models/post.rb

```diff
@@ -82,6 +82,14 @@ class Post < ApplicationRecord
     where "MONTH(favorite_lists.created_at) = ?", Time.zone.now.strftime("%m")
   end)
 
+  scope :order_by_higher_price, (lambda do
+    left_joins(:car).order("cars.price DESC")
+  end)
+
+  scope :order_by_lower_price, (lambda do
+    left_joins(:car).order("cars.price ASC")
+  end)
+
```

Comment:
```suggestion
  scope :order_by_car_price, (lambda do |sort_type|
    left_joins(:car).order(cars: {price: sort_type})
  end)
```  
anh làm thử thế này rồi thay sort_id thành sort_type là 2 chuỗi "desc" và "asc" luôn anh, khỏi dùng 1 2 cho dài dòng

Reply:
A đổi lại rồi nhé